src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.SummaryAdmin.cshtml

```diff
@@ -61,14 +62,19 @@
             }
             @if (Model.ActionsMenu != null)
```

Comment:
`&& Model.ActionsMenu.HasItems`

Reply:
`Model.ActionsMenu.HasItems` did the trick and `HasHtml` is no longer needed! Great feedback